src/evaluation/benchmark.py

```diff
@@ -124,6 +140,10 @@ def main():
                 time_pipeline_execution(
                     "original", pipeline_file, benchmark_results_pipeline_persist
```

Comment:
I think we should add sql_flavor here as a parameter as well. I don't expect different runtimes, but we should back that up with data...